src/manifest_fenix.json

```diff
@@ -0,0 +1,70 @@
+{
+  "manifest_version": 2,
+  "name": "__MSG_extensionName__",
+  "default_locale": "en_US",
+  "version": "15",
+  "author": "Firefox",
+  "applications": {
+    "gecko": {
+      "id": "secure-proxy@mozilla.com",
+      "strict_min_version": "68.0.1",
+      "update_url": "https://private-network.firefox.com/files/updates.json"
+    }
+  },
+
+  "homepage_url": "https://private-network.firefox.com",
+
+  "description": "__MSG_extensionDescription__",
+  "browser_action": {
+    "browser_style": true,
+    "default_icon": "img/badge_off.svg",
+    "default_title": "__MSG_extensionName__",
+    "default_popup": "popup/popup.html"
+  },
+
+  "icons": {
+    "48": "img/icon.svg",
+    "96": "img/icon.svg"
+  },
+
+  "incognito": "spanning",
+
+  "permissions": [
```

Comment:
GeckoView will crash with the `captivePortal` or `identity` permission in the manifest.